### DISCUSSION TITLE
Where can I donate to help make your day a bit better?

### DISCUSSION THREAD

**Discussion by SamuelKuld** [community/community#13476] [Other Feature Feedback, Questions, & Ideas]
I've worked with it for about 2 hours and I am ready to say it's the best Visual Code Extension I've installed. Made my day just watching boilerplate get filled in like a coloring book. How can I make yours?

**Comment by suprstarrd** [comment]
What extension are you talking about? This is the GitHub feedback page

**Comment by jlord** [comment]
Hiya! Are you looking to sponsor the maintainer of a project you've used?

**Comment by ZenithClown** [comment]
Generally, all big open source project has their sponsorship button on their profile and repository page. You can check any of them. You can also search/sponsor through [GitHub Sponsor](https://github.com/sponsors) Page.
